app.py

```diff
@@ -490,10 +490,44 @@ def print_events(topic=Publisher.AUTO_TOPIC, **msg_data):
     utils.debug("%s\n\tParameters: %s" % (topic, msg_data))
 
 
+def init():
+    """
+    Initialize InVesalius.
+
+    Mostly file-system related initializations.
+    """
+    #Is needed because of pyinstaller
+    multiprocessing.freeze_support()
+
+    #Needed in win 32 exe
+    if hasattr(sys,"frozen") and sys.platform.startswith('win'):
+
+        #Click in the .inv3 file support
+        root = winreg.HKEY_CLASSES_ROOT
+        key = "InVesalius 3.1\InstallationDir"
+        hKey = winreg.OpenKey (root, key, 0, winreg.KEY_READ)
+        value, type_ = winreg.QueryValueEx (hKey, "")
+        path = os.path.join(value,'dist')
+
+        os.chdir(path)
+
+    if not inv_paths.USER_INV_DIR.exists():
+        inv_paths.create_conf_folders()
+        if inv_paths.OLD_USER_INV_DIR.exists():
+            inv_paths.copy_old_files()
+
+    if hasattr(sys,"frozen") and sys.frozen == "windows_exe":
+        # Set system standard error output to file
+        path = inv_paths.USER_LOG_DIR.join("stderr.log")
+        sys.stderr = open(path, "w")
+
+
 def main():
```

Comment:
I think it's a good idea to add a parameter to indicate if is necessary to `parse_command_line`. `parse_command_line` is used to call invesalius from command line passing parameters like `-i ~/dicom/folder` to start invesalius with that dicom images already opened.

Reply:
I resolved the merge conflict. However, after some thinking, I ended up not doing the change you proposed: the reason for that is that we might want to use command line arguments with our outside wrapper for InVesalius, as well, and in that case it would be useful if our command line arguments were propagated to modify the operation of InVesalius (which happens as a default). That is, we might want to do something like:

```
python wrapper.py --no-gui
```

to start InVesalius without the GUI (just an example, there seems to be other command line arguments that might also be useful for us).

After resolving the merge conflict, this patch is now up for a re-review.